Dockerfile

```
@@ -12,11 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM alpine:3.8
+FROM alpine:3.15.6
 MAINTAINER Gardener Project
 
-RUN echo "@community http://dl-4.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories && \
-    apk add --update nodejs nodejs-npm sed curl pwgen runit@community && \
+RUN echo "@community https://dl-cdn.alpinelinux.org/alpine/v3.15/community" >> /etc/apk/repositories && \
+    apk add --update nodejs npm sed curl pwgen runit && \
```

```suggestion
RUN apk add --update nodejs npm sed curl pwgen runit && \
```

For 3.15.6 this is working too, because all packages are part of `main`.